src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingWareHouse.java

```diff
@@ -525,6 +537,8 @@ else if (request != null)
          */
         private boolean reachedLimit = false;
 
+        private List<BlockPos> storageracks = new ArrayList<>();
```

Comment:
our buildings already have a list of containers, called containerList. So you don't need to add all this stuff

Reply:
This is in the deserialize which is accessed by the View. i can not access the containerList there.. i tried a lot to access the container list but failed.

Reply:
oh its in the view, then you probably don't need it at all. The warehouse tileentity has a combined inventory of all rakcs

Reply:
oO okay ? this i built separately in the View. it iterates through all racks and adds up.

Reply:
ye you made more effort than you need to xD

Reply:
sth like tileentity.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY,null) and you get an already combined IItemhandler

Reply:
Yep this will need to be removed for simplicities sake.

Reply:
I think how it is right now it is more efficient since he gets the cached content list @OrionDevelopment